template-compose/app/src/main/java/co/nimblehq/template/compose/ui/theme/Theme.kt

```diff
@@ -1,19 +1,66 @@
 package co.nimblehq.template.compose.ui.theme
 
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.lightColors
-import androidx.compose.runtime.Composable
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.*
+import androidx.compose.runtime.*
 
 @Composable
 fun ComposeTheme(
+    darkTheme: Boolean = isSystemInDarkTheme(),
     content: @Composable () -> Unit
 ) {
-    MaterialTheme(
-        colors = lightColors(
-            primary = Color.GreenCitrus,
-            primaryVariant = Color.GreenChristi,
-            secondary = Color.GreenCitrus
-        ),
-        content = content
-    )
+    val colors = if (darkTheme) {
+        DarkColorPalette
+    } else {
+        LightColorPalette
+    }
+    val dimensions = LocalAppDimensions.current
+    val styles = LocalAppStyles.current
+    val typography = LocalAppTypography.current
+    val shapes = Shapes()
```

Comment:
Just wondering, what's the reason we did not create `LocalAppShapes` 💭

Reply:
@Tuubz Good point, I missed this. Fixed in https://github.com/nimblehq/android-templates/pull/376/commits/de566a4651372ecb658b84c946b043bcc316e1be